README.md

```diff
@@ -1 +1,71 @@
-# meals-world
\ No newline at end of file
+# Egyptian Food World
+
+## About
+
+An "Egyptian food world" helps fetch the Egyptian meals list from Meal DB API and functionality to add likes for each one and share comments using Involvement API. This website was built with ES6, Webpack, fetching the data from REST API, and unit testing using Jest.
+
+It was built upon two independent APIs:
+- [Meal DB](https://www.themealdb.com/api.php): data about meals
+- [Involvement](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270): Microverse API to record the different user interactions (likes, comments).
+
+![Screenshot1](/src/screencapture-1.png)
+![Screenshot2](/src/screencapture-2.png)
+
+## Built With
+
+- HTML, CSS, JavaScript;
+- Jest, & Webpack;
+- VisualStudio Code, Git, & GitHub;
+
+## Live Demo
+
+[Live Demo Link]( https://maha-magdy.github.io/egyptian-food-world/ )
+
```

Comment:
- [ ] I'm afraid I couldn't find the video presentation on the pull request description or the readme file, please add it. 🐶